src/task/task_controller.ts

```diff
@@ -0,0 +1,116 @@
+import { Router, Request, Response, NextFunction,} from 'express';
+import Controller from '../interface/controller_interface';
+import taskModel from './task_model';
+import { createTaskSchema, changeStatusTask ,validateBody } from '../middleware/validateBody';
+import { Task } from './task_interface';
+
+class TaskController implements Controller {
+    
+    path: string = "/task";
+    router: Router = Router();
+    private task = taskModel;
+
+    constructor(){
+        this.initialiseRoutes()
+    }
+
+    private initialiseRoutes(){
+        // Add Middlewares
+        this.router.post(`${this.path}`, validateBody(createTaskSchema), this.createTask)
+        this.router.get(`${this.path}/getAll/:id`, this.getTasks)
+        this.router.put(`${this.path}/add`, this.AddMemberToTask)
+        this.router.put(`${this.path}/remove`, this.RemoveMemberFromTask)
```

Comment:
Keep the function casing consistent for all the functions and keep it in `camelCase` unlike methods, classes, types and interfaces use the Pascal case i.e. `AppleClass`

Reply:
Got it